crypto/kdf/scrypt.c

```diff
@@ -506,6 +514,10 @@ static int scrypt_alg(const char *pass, size_t passlen,
 
 const EVP_PKEY_METHOD *scrypt_pkey_method(void)
 {
+    if (FIPS_mode()) {
+        FIPSerr(ERR_LIB_FIPS, FIPS_R_NON_FIPS_METHOD);
+        return NULL;
+    }
     return &scrypt_pkey_meth;
```

Comment:
We might want to leave scrypt allowed. See my comment above.

Reply:
No, it's not listed in our security policy so we must disable it. Jason can follow-up here if needed.